concent_api/core/views.py

```diff
@@ -716,6 +757,33 @@ def handle_receive_out_of_band_reject_report_computed_task(request, undelivered_
     return message_verdict
 
 
+def handle_receive_out_of_band_force_subtask_results(
```

Comment:
This function and `handle_receive_delivered_force_subtask_results` only differs by status passed to `store_message_and_message_status`. Please combine them into single function with status as parameter.

Reply:
:+1: